data/mods/Xedra_Evolved/procgen/dreamsmith_procgen.json

```diff
@@ -212,79 +212,6 @@
     "id": "dreamsmith_lottery_def_head",
     "//": "In addition to stats and armor, this pool should related to perception, sight, or vision",
     "passive_add_procgen_values": [
```

Comment:
Artifact code do not support this values, it can't be added back, nor i really want to add it back because it's impossible to make player aware of them without heavy code alteration

Reply:
The player not (yet) being aware of them, applies to the other dreamsmithing enhancements too (which you didn't emove), and it appears that @Standing-Storm is interested in eventually displaying them (see https://github.com/CleverRaven/Cataclysm-DDA/pull/76957, under Additional context). Also, dreamsmith armor not being extra protective doesn't fit well with the dreamsmithing theme.

(Also, now I see ITEM_ARMOR_*, I think those are a better fit than ARMOR_* -- unless some fancy magic is going on (might be the case), then why would, say, head armor protect the feet? Seems much nicer than ARMOR_*.)

Reply:
I agree that on it's own it's a nice concept, i may try to do something about them being visible; 
Even with enchantments, armor was not really "extra protective", rather it just granted some specific armor against specific damage type
I would like to hear if Storm is interested in expanding the pool of possible enchantments for relics from Storm itself
ITEM_ARMOR_ is also removed, and without replacement, because it couldn't work properly, and was not used overall